python_modules/dagster/dagster_tests/cli_tests/command_tests/test_launch_command.py

```diff
@@ -255,3 +255,23 @@ def test_job_launch_handles_pipeline():
 
         # dont care if its a pipeline and not a job
         execute_launch_command(instance, pipeline_kwargs)
+
+
+def test_launch_command_help():
+    runner = CliRunner()
+    result = runner.invoke(job_launch_command, ["--help"])
+    print(str(result.stdout))
+
+    assert (
+        """  -m, --module-name TEXT        Specify module or modules (flag can be used
+                                multiple times) where dagster definitions reside
+                                as top-level symbols/variables and load each
+                                module as a code location in the current python
+                                environment.
+  -f, --python-file PATH        Specify python file or files (flag can be used
+                                multiple times) where dagster definitions reside
+                                as top-level symbols/variables and load each
+                                file as a code location in the current python
+                                environment."""
+        in result.stdout
+    )
```

Comment:
would support searching for a smaller substring here e.g. `assert "flag can be used multiple times" in result.stdout` so this doesn't fail on whitespace-only changes and the like